### PR TITLE
CI 1.36 and 1.37 RN for CI.

### PR DESCRIPTION
**Pull request by dewan-ahmed**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: CI 1.36 and 1.37 Release Notes
* Preview links/images (Internal contributors only): \__________________

## Merge Requirements

- [x] Successful preview build.
- [ ] Code owner review.
- [x] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Comment by dewan-ahmed** [comment]
These release notes are still being worked on. Please **do not merge**.

Once the PR is ready, I'll remove the **do not merge** label and this PR will go through PM review and editorial review.

**Review by lkilpatrick-harness — APPROVED** [review]
LGTM

File: release-notes/continuous-integration.md [hunk 1]
```diff
@@ -37,6 +37,46 @@ Contact [Harness Support](mailto:support@harness.io) if you have any questions.
 
 ## July 2024
 
+### Version 1.37
+
+<!-- 2024-07-16 -->
+
+#### Early Access feature
```
Comment (nofarblue) [inline release-notes/continuous-integration.md:44]: Can you take the git clone out of this release notes ?
it goes in in 1.39

**Review by nofarblue — CHANGES_REQUESTED** [review]

File: release-notes/continuous-integration.md (hunk 1 above)
Comment (dewan-ahmed) [reply release-notes/continuous-integration.md]: sure. resolved via https://github.com/harness/developer-hub/pull/7464/commits/b1457728b4c3c1b18734ec2b33c85f573afdcfd9

**Comment by dewan-ahmed** [comment]
@nofarblue please review.

**Review by nofarblue — APPROVED** [review]

**Pull request merged**